snapcraft/internal/lxd.py

```diff
@@ -67,7 +67,11 @@ def _create_container(self):
                 self._container_name])
             yield
         finally:
-            check_call(['lxc', 'stop', self._container_name])
+            # We do this as lxc stop does not print anything out as
+            # it does when creating the container and stopping can take
+            # a bit.
```

Comment:
This text is confusing. Maybe just: # Stopping takes a while and lxc doesn't print anything.